nautobot/extras/factory.py

```diff
@@ -167,6 +167,15 @@ def jobs(self, create, extracted, **kwargs):
             job.validated_save()
 
 
+class JobQueueFactoryWithTenancy(JobQueueFactory):
+    class Meta:
+        model = JobQueue
+
+    class Params:
+        has_description = NautobotBoolIterator()
+        has_tenant = NautobotBoolIterator(chance_of_getting_true=100)
```

Comment:
Bad merge conflict? This isn't relevant to your PR is it?

Reply:
This either was me trying to fix an issue with JobQueue tests that often failed due to missing Tenant filter, or a bad merge. I don't remember so I will remove.